src/requirements.txt

```diff
@@ -1,61 +1,61 @@
 -i https://pypi.python.org/simple
 annotated-types==0.6.0; python_version >= '3.8'
 asgiref==3.7.2; python_version >= '3.7'
-boto3==1.33.7; python_version >= '3.7'
-botocore==1.33.7; python_version >= '3.7'
+boto3==1.34.37; python_version >= '3.8'
+botocore==1.34.37; python_version >= '3.8'
 cachetools==5.3.2; python_version >= '3.7'
-certifi==2023.11.17; python_version >= '3.6'
+certifi==2024.2.2; python_version >= '3.6'
 cfenv==0.5.3
-cffi==1.16.0; python_version >= '3.8'
+cffi==1.16.0; platform_python_implementation != 'PyPy'
 charset-normalizer==3.3.2; python_full_version >= '3.7.0'
-cryptography==41.0.7; python_version >= '3.7'
+cryptography==42.0.2; python_version >= '3.7'
 defusedxml==0.7.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 dj-database-url==2.1.0
 dj-email-url==1.0.6
-django==4.2.7; python_version >= '3.8'
+django==4.2.3; python_version >= '3.8'
 django-allow-cidr==0.7.1
 django-auditlog==2.3.0; python_version >= '3.7'
 django-cache-url==3.4.5
 django-cors-headers==4.3.1; python_version >= '3.8'
 django-csp==3.7
 django-fsm==2.8.1
 django-login-required-middleware==0.9.0
-django-phonenumber-field[phonenumberslite]==7.2.0; python_version >= '3.8'
+django-phonenumber-field[phonenumberslite]==7.3.0; python_version >= '3.8'
 django-widget-tweaks==1.5.0; python_version >= '3.8'
-environs[django]==9.5.0; python_version >= '3.6'
-faker==20.1.0; python_version >= '3.8'
+environs[django]==10.3.0; python_version >= '3.8'
+faker==23.1.0; python_version >= '3.8'
 fred-epplib@ git+https://github.com/cisagov/epplib.git@d56d183f1664f34c40ca9716a3a9a345f0ef561c
 furl==2.1.3
 future==0.18.3; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
 gevent==23.9.1; python_version >= '3.8'
-geventconnpool@ git+https://github.com/rasky/geventconnpool.git@1bbb93a714a331a069adf27265fe582d9ba7ecd4
-greenlet==3.0.1; python_version >= '3.7'
+geventconnpool@ git+https://github.com/rasky/geventconnpool.git
```

Comment:
needs to be reverted to what it is on main

Reply:
done